src/ontology/nbo-edit.owl

```diff
@@ -9938,6 +9942,17 @@ AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0050002 "behavior_ontology"
 AnnotationAssertion(rdfs:label obo:NBO_0050002 "circling behavior")
 SubClassOf(obo:NBO_0050002 obo:NBO_0000013)
 
+# Class: obo:NBO_0070000 (autotomy)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI 10.1093/beheco/arl010") obo:IAO_0000115 obo:NBO_0070000 "Protective behavior that sheds or discards an animal's body part.")
```

Comment:
```suggestion
AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1093/beheco/arl010") obo:IAO_0000115 obo:NBO_0070000 "Protective behavior that sheds or discards an animal's body part.")
```

Reply:
(Xrefs should be valid CURIEs. This way, you can easily turn them into URLs of the type:

https://doi.org/10.1093/beheco/arl010